Makefile

```diff
@@ -12,7 +12,7 @@ buildkite-plugin-lint:
 
 shellcheck:
 	@echo ~~~ 🕵️ ShellCheck
-	$(docker_run) koalaman/shellcheck hooks/** bin/** --exclude=SC1071
+	$(docker_run) koalaman/shellcheck $(shell find hooks bin -type f -not -name "*.ps1") --exclude=SC1071
```

Comment:
👍 

(I wonder if there's a `shellcheck` equivalent for PowerShell btw… though probably only a thing to look at when we'll start tacking #101 more officially/broadly?)

Reply:
> I wonder if there's a shellcheck equivalent for PowerShell btw…

Found [this](https://www.powershellgallery.com/packages/PSScriptAnalyzer/)… in case we want to look into it at some point